lib/components/form/batch-settings.tsx

```diff
@@ -100,7 +114,9 @@ class BatchSettings extends Component<{
 }> {
   state = {
     expanded: null,
-    selectedModes: this.props.modeOptions.map((m) => m.mode)
+    selectedModes: this.props.modeOptions
+      .filter((m) => !m.defaultUnselected)
+      .map((m) => m.mode)
```

Comment:
Use new method `getDefaultModes` here too? (this block looks very similar)

Reply:
Is this block a new change? I can't get it to pick up the unselected modes. Should probably be moved to the class constructor.

Reply:
The bug was a result of a merge conflict that we fixed in 647708e50a79e9185017b051388ff377b214dd83. We've also used the new method in 6ffad50f66e885e0f7db863d42cfe857bc14291b